iothub/device/src/Authentication/IotHubConnectionCredentials.cs

```diff
@@ -132,6 +132,13 @@ public IotHubConnectionCredentials(string iotHubConnectionString)
         /// <summary>
         /// The full chain of certificates from the one used to sign the client certificate to the one uploaded to the service.
         /// </summary>
+        /// <remarks>
+        /// If supplied, the certificate chain will be verified and sent to the service in the request by adding them to the current user
+        /// intermediate certificate store.
```

Comment:
Nit: "Intermediate Certification Authorities" store.
Optionally we can also point to https://learn.microsoft.com/en-us/troubleshoot/developer/webapps/iis/www-authentication-authorization/configure-intermediate-certificates (or similar).